master/Makefile

```diff
@@ -46,13 +46,13 @@ check-gen: force-gen gen
 
 .PHONY: get-deps
 get-deps:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint
-	go install golang.org/x/tools/cmd/goimports
-	go install github.com/goreleaser/goreleaser
-	# For hyperparameter importance computation
-	go get github.com/ryanbressler/CloudForest
-	go install github.com/ryanbressler/CloudForest/growforest
-	go get github.com/vektra/mockery/v2/.../@v2.9.2
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.43.0
```

Comment:
I'm surprised that we didn't already move to the latest when we weren't specifying a version. Am I missing something?

Reply:
yeah we were, it was pinned in the `go.mod` and imported into `tools.go` so it was "used"